harness/src/utils.rs

```diff
@@ -128,6 +130,22 @@ pub fn kill_process_using_port(port: u16) {
         .expect("Failed to execute kill command");
 }
 
+pub async fn count_voter(raft: &Raft) -> usize {
+    let mut hash_set = HashSet::new();
+    let start_time = Instant::now();
+    while start_time.elapsed() < Duration::from_secs(1) {
+        raft.get_peers()
+            .await
+            .inner
+            .iter()
+            .filter(|(_, role)| role.role == InitialRole::Voter)
+            .for_each(|(id, _)| {
+                hash_set.insert(*id);
+            });
+    }
+    hash_set.len()
+}
+
 pub fn kill_previous_raft_processes() {
```

Comment:
Is there a reason why the counting needs to be executed multiple times for 1 second, rather than just once after a 1-second delay?